### ISSUE TITLE
Original have torch 1.5.0 and when install PyTorch Geometric, it automatically downgr

### ISSUE THREAD

**Issue by xdwang0726** [rusty1s/pytorch_geometric#1327]
## Environment

* OS:
* Python version:
* PyTorch version:
* CUDA/cuDNN version:
* GCC version:
* How did you try to install PyTorch Geometric and its extensions (pip, source):
* Any other relevant information:

## Checklist

- [ ] I followed the installation guide.
- [ ] I cannot find my error message in the FAQ.
- [ ] I set up CUDA correctly and can compile CUDA code via `nvcc`.
- [ ] I have cloned the repository and tried a manual installation from source.
- [ ] I do have multiple CUDA versions on my machine.
- [ ] I checked if the official extension example runs on my machine.
- [ ] The offical extension example runs on my machine.

**Comment by rusty1s** [comment]
That is weird. PyG does not re-install PyTorch, and I see no particular reason why that should happen. Are you sure you don‘t have multiple PyTorch versions installed?

**Comment by xdwang0726** [comment]
> That is weird. PyG does not re-install PyTorch, and I see no particular reason why that should happen. Are you sure you don‘t have multiple PyTorch versions installed?

I only have one PyTorch version. There is the version after install PyG. 
```
(pytorch) [xdwang@beluga2 ~]$ python -c "import torch; print(torch.__version__)"
1.4.0+cu100
```
Before it was 1.5.0

**Comment by rusty1s** [comment]
Well, you can still always upgrade to PyTorch 1.5 afterwards. Regarding your error, you may want to make sure that libcusparse exists in /usr/local/cuda/lib.

**Comment by xdwang0726** [comment]
> Well, you can still always upgrade to PyTorch 1.5 afterwards. Regarding your error, you may want to make sure that libcusparse exists in /usr/local/cuda/lib.

Right, I saw the solution to the import error on the other issue. As I am using a cluster which I cannot access to the CUDA path. I have contacted the support team on my cluster and see if ```libcusparse``` is the problem. Will come back and close the issue if works. Thank you!

**Comment by yulonglin** [comment]
PyTorch is also reinstalled when I run `!pip install --target=$lib_path torch-geometric` on Google Colab, where I'm trying to install it to a specific path `lib_path`. It upgrades `torch` (from `1.7.0` to `1.7.1` and a few other things into incompatible versions.

**Comment by yulonglin** [comment]
PyTorch is also reinstalled when I run !pip install --target=$lib_path torch-geometric on Google Colab, where I'm trying to install it to a specific path lib_path. It upgrades torch (from 1.7.0 to 1.7.1) and a few other things into incompatible versions.

**Comment by yulonglin** [comment]
PyTorch is also reinstalled when I run `!pip install --target=$lib_path torch-geometric` on Google Colab, where I'm trying to install it to a specific path `lib_path`. It upgrades `torch` (from `1.7.0` to `1.7.1`) and a few other things into incompatible versions.

**Comment by rusty1s** [comment]
PyG will install `torch==1.7.1+cu102` in case no PyTorch version is not found. I guess you are save to delete this package and install the PyTorch package that works for you.

**Comment by yulonglin** [comment]
Ok sure, yes I parsed the incompatibility error message from installing PyG and deleted the corresponding packages and it works fine now.

**Comment by CharlesGaydon** [comment]
Adding my two cents: 
I had `pytorch==1.8.0` installed, and running `conda install pyg -c pyg -c conda-forge` would downgrade it to `pytorch=1.10.0`. 
My solution was to choose a specific build from the [conda list of files for pyg package](https://anaconda.org/pyg/pyg/files?version=2.0.2), and to run something like:
`mamba install pyg=2.0.2=py39_torch_1.9.0_cu111 -c pyg -c conda-forge`
